CONTRIBUTING.md

```diff
@@ -91,4 +91,45 @@ Naming methods properly is important for multiple reasons:
 
 `method's name` + `type
 
-where `type` is optional since in some cases it could be inferred from the method/class name itself. E.g. `RegressorChain`
\ No newline at end of file
+where `type` is optional since in some cases it could be inferred from the method/class name itself. E.g. `RegressorChain`
+
+### Template code
+
+```python
+   
+# Import required packages such as estimators, generators, etc
+from skmultiflow.data import SEAGenerator
+from skmultiflow.lazy import KNNClassifier
+
+# Setup a data stream
+stream = SEAGenerator(random_state=1)
+
+# Prepare stream for use
+stream.prepare_for_use()
+
+# Variables to control loop and track performance
+n_samples = 0
+correct_cnt = 0
+max_samples = 200
+X, y = stream.next_sample(max_samples)
+
+# Setup the desired estimator
+estimator = KNNClassifier(n_neighbors=8,
+                          max_window_size=2000,
+                          leaf_size=30)
+
+# Pre train the estimator with the max samples specified
```

Comment:
The template should not include pre-training. Please replace the estimator with the `NaiveBayesClassifier` and remove the pre-training code.